src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -81,6 +81,15 @@ public void Configure(TelemetryConfiguration configuration)
             {
                 module.Initialize(configuration);
             }
+
+            foreach (ITelemetryProcessor processor in configuration.TelemetryProcessors)
+            {
+                ITelemetryModule module = processor as ITelemetryModule;
```

Comment:
@dnduffy Not really, as Paul alluded below, this change is intended for telemetry processors that also require initialization (that is why they implement ITelemetryModule). They won't be _registered_ as telemetry modules.